tests/baselines/reference/ModuleWithExportedAndNonExportedFunctions.errors.txt

```diff
@@ -1,4 +1,4 @@
-tests/cases/conformance/internalModules/exportDeclarations/ModuleWithExportedAndNonExportedFunctions.ts(28,13): error TS2339: Property 'fn2' does not exist on type 'typeof A'.
+tests/cases/conformance/internalModules/exportDeclarations/ModuleWithExportedAndNonExportedFunctions.ts(28,13): error TS2551: Property 'fn2' does not exist on type 'typeof A'. Did you mean 'fng'?

```

Comment:
Ah this is the same reason why the other result @RyanCavanaugh pointed out appeared, too.